api/Element.json

```diff
@@ -1964,43 +1964,43 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/innerHTML",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": true
             },
             "chrome": {
-              "version_added": null
+              "version_added": "1"
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": "18"
             },
             "edge": {
-              "version_added": null
+              "version_added": "12"
             },
             "edge_mobile": {
-              "version_added": null
+              "version_added": "12"
             },
             "firefox": {
-              "version_added": null
+              "version_added": "1"
             },
             "firefox_android": {
-              "version_added": null
+              "version_added": "4"
             },
             "ie": {
-              "version_added": null
+              "version_added": "4"
             },
             "opera": {
-              "version_added": null
+              "version_added": "7"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": true
```

Comment:
As Safari for iOS supported this from the start, the same therefore applies to desktop Safari, therefore this should probably be `"1"`.

Reply:
Not 100% sure: iPhone OS 1 has been released on June 2008, whereas Safari 1 has been released on June 2003.

For perspective, Opera 7, the first Opera version to support innerHTML, has been released on January 2003.

Reply:
Because Safari for iOS was released with the iPhone in 2007 when it was still called "iPhone OS", it's hard to really determine what the version numbers should be for early versions, but I'm pretty sure it shouldn't be "1".

Reply:
Lets go with true for now.